rotkehlchen/api/rest.py

```diff
@@ -4279,7 +4280,11 @@ def _get_ens_mappings(
                         mappings_to_send[cached_value.address] = cached_value.name  # type: ignore
                 addresses_to_query += list(set(addresses) - set(cached_data.keys()))
 
-            query_results = self.rotkehlchen.chain_manager.ethereum.ens_reverse_lookup(addresses_to_query)  # noqa: E501
+            try:
+                query_results = self.rotkehlchen.chain_manager.ethereum.ens_reverse_lookup(addresses_to_query)  # noqa: E501
+            except (RemoteError, BlockchainQueryError) as e:
+                msg = f'Error occurred while querying ens names: {str(e)}'
+                return wrap_in_fail_result(message=msg, status_code=HTTPStatus.CONFLICT)
```

Comment:
This would probably need a check with the frontend if they are handling it properly. @lukicenturi are you guys handling 409 for this endpoint properly?